alf/summary/summary_ops.py

```diff
@@ -19,6 +19,17 @@
 from torch.utils.tensorboard import SummaryWriter
 from typing import Callable
 
+try:
+    # If tensorflow has been installed, pytorch might use tensorflow's
+    # tensorboard. In this case, gfile needs to be redirected if embedding
+    # projector is to be used.
+    # https://github.com/pytorch/pytorch/issues/30966#issuecomment-582747929
+    import tensorflow as tf
+    import tensorboard as tb
+    tf.io.gfile = tb.compat.tensorflow_stub.io.gfile
```

Comment:
Someone in that thread said that this fix "did slow down program execution by a huge factor if you have many summarywriter calls"

Reply:
I think that it doesn't make sense to call many times. In that case, there will be a long list of figures, each runs visualization. For most applications, summarize several times throughout the training will be good enough.

Reply:
But this fix seems affecting all the summarywriter calls.

Reply:
Not sure about if that person meant only for embedding summary or all summaries, because the gfile error only occurs when calling add_embedding(). I can do a test

Reply:
Seems that only add_embedding uses gfile for file I/O. Other summary calls use TB's FileWriter. I just tested and this change won't affect the efficiency if add_embedding is not called many times.